test/requests.js

```diff
@@ -703,6 +705,78 @@ var tests = function(web3) {
       });
     });
 
+    // NB: relies on the previous test setting value to 25 and the contract deployment setting
+    // original value to 5. `contractCreationBlockNumber` is set in the first test of this
+    // describe block.
+    it("should read data via a call at a specified blockNumber (eth_call)", function(done){
+      var startingBlockNumber = null;
+      var call_data = contract.call_data;
+
+      web3.eth.getBlockNumber().then(function(result){
+
+        startingBlockNumber = result;
+        return web3.eth.call(call_data)
+
+      }).then(function(result){
+
+        assert.equal(to.number(result), 25, "value retrieved from latest block should be 25");
+        return web3.eth.call(call_data, contractCreationBlockNumber)
+
+      }).then(function(result){
```

Comment:
Is this too spacey?

Reply:
doesn't bother me any – generally I'm pro whitespace